core/frontend/src/RealityDataSourceTilesetUrlImpl.ts

```diff
@@ -105,9 +110,17 @@ export class RealityDataSourceTilesetUrlImpl implements RealityDataSource {
     return true;
   }
 
-  /** Returns the tile URL. If the tile path is a full URL, it is returned as is. Otherwise, the base URL is prepended to the tile path. */
+  /** Returns the tile URL. If the tile path is a full URL, the URL's search parameters are checked.
+   * If the search parameters are empty, the root url's search parameters will be appended to the tile path.
+   * Otherwise, the base URL is prepended to the tile path, and the root url's search parameters are appended,
+   * if no search parameters exist on the existing tile path.
+   */
```

Comment:
```suggestion
  /** Returns the tile URL.
   * If the tile path is a relative URL, the base URL is prepended to it.
   * For both absolute and relative tile path URLs, the search parameters are checked. If the search params are empty, the base URL's search params are appended to the tile path.
   */
```
Suggestion to simplify the description